.github/workflows/javaTests.yml

```diff
@@ -85,6 +85,7 @@ jobs:
           "**.functions.reorg.**,**.functions.rewrite.**,**.functions.ternary.**",
           "**.functions.transform.**","**.functions.unique.**",
           "**.functions.unary.matrix.**,**.functions.linearization.**,**.functions.jmlc.**"
+          "**.functions.reorg.**,**.functions.rewrite.**,**.functions.ternary.**"
```

Comment:
this looks like a duplicate of line 85, therefore only double the number of times we run the test.

Reply:
Thanks for letting me know.